scripts/jtsync/jtsync.rb

```diff
@@ -57,6 +57,9 @@ def version
     end
 
     def card_name
+      if name.end_with?("mediacheck")
+        @name = name.gsub("cloud-", "")
+      end
       "C#{version} #{name}"
     end
```

Comment:
I rather meant something like this:
```
CARDNAMES = {
  "cloud-mediacheck" => "mediacheck"
}

def card_name
  cname = CARDNAMES[name] || name
  "C#{version} #{cname}"
end
```

Reply:
with this adding a new special case is only one entry to the mapping, instead of a new if or switch block

Reply:
I changed it. Sorry for the long delay.